MvvmCross/Binding/Droid/Views/MvxExpandableListView.cs

```diff
@@ -151,7 +151,11 @@ private void EnsureGroupClickOverloaded()
 
         private void GroupOnClick(object sender, GroupClickEventArgs e)
         {
-           ExecuteCommandOnGroup(GroupClick, e.GroupPosition);
+            if (IsGroupExpanded(e.GroupPosition))
```

Comment:
Is this change really needed, don't we just need to set `GroupClickEventArgs.Handled` property to true or false to signal whether we handled the event?

Reply:
Tried to find a place to set that property but I could not find one as the args are already in there when the method is getting called. By looking around I saw that there is a difference over how we use ItemClick and GroupClick, but will look more into it tomorrow. (GroupClick exists in both base and Mvx when ItemClick exists only in Mvx, while we have ChildClick in base, and maybe something goes wrong around this?) Any other idea?

Reply:
Just as I suggested, it is enough to just add `e.Handled = false;` after `ExecuteCommandOnGroup(..)`.